dashboard/apps/dashboard/spreadsheets.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+from datetime import date
+import re
+
+from openpyxl.styles import Style, Font
+from openpyxl.workbook import Workbook
+
+from dashboard.libs.date_tools import parse_date
+
+
+class Products:
+
+    date_style = Style(number_format='DD/MM/YYYY')
+    bold_font = Font(bold=True)
+    bold_style = Style(font=bold_font)
+    currency_style = Style(number_format='£#,##0.00')
```

Comment:
IMO this decision can be deferred to whenever a new class gets added to the module. `bold_style` is probably universal but `date_style` may not be that reusable.